packages/common/src/content/_internal.ts

```diff
@@ -183,6 +183,8 @@ const getSolutionUrl = (
       // deployed solutions go to the generic content route
       hubUrl = `/content/${identifier}`;
     }
+    // TODO: I think this is a bug. The line below should be w/in else
+    // in order to avoid overwriting the above for deployed solutions, right?
```

Comment:
I think you are correct @tomwayson

Reply:
thx. I wonder if we have some code in -ui that is correcting for this bug. Maybe @vivzhang knows?